src/main/java/net/dv8tion/jda/api/entities/ThreadChannel.java

```diff
@@ -140,6 +140,47 @@ default GuildMessageChannel getParentMessageChannel()
         throw new UnsupportedOperationException("Parent of this thread is not a MessageChannel. Parent is type: " + getParentChannel().getType().getId());
     }
 
+    /**
+     * Attempts to get the {@link net.dv8tion.jda.api.entities.Message Message} from Discord's servers that started this thread.
+     *
+     * <p>The {@link Message#getMember() Message.getMember()} method will always return null for the resulting message.
+     * To retrieve the member you can use {@code getGuild().retrieveMember(message.getAuthor())}.
```

Comment:
I know this comes from our other docs related to retrieveMessage, but I question it.
@MinnDevelopment is this actually valid anymore? Stepping through the logic 
 - [MessageChannel#retrieveMessageById uses EntityBuilder#createMessageWithChannel](https://github.com/DV8FromTheWorld/JDA/blob/b46ea213f3f6ac6daaf764b045a6aacd6e9865d1/src/main/java/net/dv8tion/jda/api/entities/MessageChannel.java#L839)
  - [EntityBuilder#createMessageWithChannel uses #createMessage0](https://github.com/DV8FromTheWorld/JDA/blob/d7bdd6c5f6ddddd5fa53a5853fde17f13bd3f163/src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java#L1331)
  - [EntityBuilder#createMessage0 doesn't build the member because the json doesn't container 'member'](https://github.com/DV8FromTheWorld/JDA/blob/d7bdd6c5f6ddddd5fa53a5853fde17f13bd3f163/src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java#L1401)
  - [EntityBuilder#createMessage0 attaches member information if it didn't have it but it exists in the cache](https://github.com/DV8FromTheWorld/JDA/blob/d7bdd6c5f6ddddd5fa53a5853fde17f13bd3f163/src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java#L1438-L1439)
  
  It seems like if Member is cached then a Rest retrieved message could have a member attached to it.

Reply:
It is unreliable, so i just documented as null. Documenting all cache edge cases seems overkill.